moved.tf

```diff
@@ -14,21 +18,41 @@ moved {
   to   = module.logdna.ibm_resource_instance.logdna
 }
 
+moved {
+  from = module.logdna.ibm_resource_instance.logdna
+  to   = module.log_analysis.ibm_resource_instance.log_analysis
+}
+
 moved {
   from = ibm_resource_key.log_dna_resource_key
   to   = module.logdna.ibm_resource_key.resource_key
 }
 
+moved {
+  from = module.logdna.ibm_resource_key.resource_key
+  to   = module.log_analysis.ibm_resource_key.resource_key
+}
+
 moved {
   from = ibm_resource_instance.sysdig
   to   = module.sysdig.ibm_resource_instance.sysdig
 }
 
+moved {
+  from = module.sysdig.ibm_resource_instance.sysdig
+  to   = module.cloud_monitoring.ibm_resource_instance.cloud_monitoring
+}
+
```

Comment:
Yes, as per my understanding this is the best way to handle this since I initially implemented the moved block in the same way as you are suggesting and tested it. It was destroying resources if I did `terraform plan` on my code after doing `terraform apply` in the main code suggesting that the moved block was not working properly that way.